index.html

```diff
@@ -86,6 +86,7 @@ <h2>Participants</h2>
         <a target="_blank" href="participants/verity-brean/">Verity Brean</a>
         <a target="_blank" href="participants/yael-steinberger/">Yael Steinberger</a>
         <a target="_blank" href="participants/zain-afzal/">Zain Afzal</a>
+        <a target="_blank" href="participants/ritvik-mathur/">Ritvik Mathur</a>
```

Comment:
i can't seem to find your folder in the `participants` folder. Are you sure you've added it?